src/command_modules/azure-cli-netappfiles/azure/cli/command_modules/netappfiles/_params.py

```diff
@@ -0,0 +1,60 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+# pylint: disable=line-too-long
+from azure.cli.core.commands.parameters import tags_type
+
+
+def load_arguments(self, _):
+    with self.argument_context('netappfiles') as c:
+        c.argument('resource_group', options_list=['--resource-group', '-g'], required=True, help='The name of the resource group')
+        c.argument('tags', arg_type=tags_type)
+
+    with self.argument_context('netappfiles') as c:
+        c.argument('account_name', options_list=['--account-name', '-a'], required=True, help='The name of the ANF account')
```

Comment:
`required=True/False` should not be necessary anywhere in this file. It is inferred from the command signatures.

Reply:
ok, removed